{"name": "test_yaml[login1]", "status": "passed", "attachments": [{"name": "log", "source": "cbf5729a-3dc2-4885-a3bc-c2a4e905aa92-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "login", "value": "{'case_name': '登录失败用例', 'url': '/authorizations/', 'data': {'username': 'test123456', 'password': '111111'}, 'expect': '\"user_id\": 1, \"username\": \"python\"'}"}], "start": 1586788691666, "stop": 1586788691987, "uuid": "1313cf7e-6fe7-48b7-b962-f577a84e23c4", "historyId": "0f3f31b3b063cf542745ec9591837385", "testCaseId": "c952dcecfc445943309748bf6fc52050", "fullName": "testcase.Test_login#test_yaml", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "Test_login"}, {"name": "host", "value": "LAPTOP-8FSK7C3Q"}, {"name": "thread", "value": "15992-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.Test_login"}]}